apps/microtvm/zephyr/template_project/CMakeLists.txt.template

```diff
@@ -31,21 +31,27 @@ find_package(Zephyr HINTS $ENV{ZEPHYR_BASE})
 project(microtvm_autogenerated_project)
 
 if(DEFINED CMSIS_PATH)
+  if (EXISTS ${CMSIS_PATH}/CMSIS-NN)
```

Comment:
Is this a temporary fix until we update the docker image?

Reply:
Yes it is.